src/api/github/webhooks/index.ts

```diff
@@ -0,0 +1,10 @@
+const { createNodeMiddleware, createProbot } = require("probot");
+
+const app = require("../../../app");
+const probot = createProbot({
+  defaults: {
+    webhookPath: "/api/github/webhooks",
+  },
+});
+
+module.exports = createNodeMiddleware(app, { probot });
```

Comment:
why the `src/` prefix? I think Vercel supports TypeScript out of the box, and the webhook handler function needs to be in the `api/` folder.

Reply:
I still have pending the deploy part. For the moment I just removed this file from the draft PR. And, once validated, I will focus on learning how vercel deployment for probot works :)